nsd-control.c

```diff
@@ -521,8 +522,11 @@ int main(int argc, char* argv[])
 	if(argc == 0)
 		usage();
 	if(argc >= 1 && strcmp(argv[0], "start")==0) {
-		if(execl(NSD_START_PATH, "nsd", "-c", cfgfile, 
-			(char*)NULL) < 0) {
+		const char *path;
+		if((path = getenv("NSD_PATH")) == NULL) {
+			path = NSD_START_PATH;
+		}
```

Comment:
The new NSD_PATH env variable, I do not recall documentation for it (or maybe it is somewhere else), and that would be good to have.

Reply:
It's needed solely to run the CreDNS test-suite that I copied and didn't document it on purpose to keep it "hidden". If I document it, it's a "supported" feature. Do we want that?

Reply:
Some hidden debug feature in our production code is unwanted.  If we can turn it into a supported feature that is useful for users, that would be good too.  Otherwise change the test setup (maybe just set $PATH or start nsd directly?)

Reply:
I mean, to answer more directly, I don't mind this as a supported feature.

Reply:
It starts `nsd` directly. `NSD_START_PATH` is defined in `config.h`, that's the problem when trying to run in the build directory (I don't want to install while developing). Guess I'll be documenting the environment variable :slightly_smiling_face:. Should I add it to the manpage for `nsd-control`?

Reply:
I mean, it is possible to start nsd by starting ../../nsd instead of ../../nsd-control start.  Then the NSD_START_PATH does not matter, right?  But adding it to the manpage is what I originally asked you for.

Reply:
Done